topo/topo.go

```diff
@@ -49,6 +49,61 @@ import (
 	_ "github.com/google/kne/topo/node/srl"
 )
 
+// Resourcer loads and reports resources of a topology.
+type Resourcer interface {
```

Comment:
all of these separate interfaces are not required it seems, was this done to increase clarity?

It might make more sense to define a single TopologyManager interface like you've done before that includes all these methods inside of it directly. I don't see a real benefit to defining these subinterfaces separately.

What do you think?

Reply:
Yes, it is just for clarity purpose. Revised the code by putting methods directly into the TopologyManager interface; we can easily create sub-interface if needed in the future.